go-controller/pkg/ovn/gress_policy.go

```diff
@@ -100,10 +100,18 @@ func (gp *gressPolicy) ensurePeerAddressSet(factory AddressSetFactory) error {
 	return nil
 }
 
-func (gp *gressPolicy) addPeerPod(pod *v1.Pod) error {
+func (gp *gressPolicy) addPeerPod(pod *v1.Pod, oc *Controller) error {
```

Comment:
Can we just shortcut to exit without an error if the annotation doesn't exist?

Reply:
seems like you and @dcbw prefer that, so I'm cool with it